### PR TITLE
improve docs on merging/splitting activities

### PR DESCRIPTION
**Pull request by GeraldJansen**
Clarify how overlapping activities are split/merged. Refs #396.

### CODE REVIEW

**Comment by ederag** [comment]
But inserting an activity into an ongoing one fails.
So this is not only a documentation issue to me.
There is a strong potential for breaking things here, so this will have to wait a little
(probably after the v2.3 release).

File: help/C/merge.page [hunk 1]
```diff
@@ -9,26 +9,18 @@
 
 <p>
     <app>Time Tracker</app> does its best to avoid overlaps in activities.
-    If you create an activity in the middle of an existing activity, the former
-    will be split into two. In other cases of overlapping, the previous entries
-    will be shrunk.
+    If you create an activity whose start and end time both lie within an
+    existing activity, the existing activity will be split into two. In
+    other cases of overlapping, the previous entries will be shrunk.
```
Comment (ederag) [inline help/C/merge.page]: OK

File: help/C/merge.page [hunk 2]
```diff
@@ -9,26 +9,18 @@
...
 </p>
-<note style="bug">
-    <title>Former activity not split</title>
-    <p>
-        The former activity is not "split into two" actually,
-        but merely shortened to end when the new activity starts.
-        Reported as <link href="https://github.com/projecthamster/hamster/issues/396">issue 396</link>.
-    </p>
-</note>
```
Comment (ederag) [inline help/C/merge.page]: This issue is not entirely solved. I would prefer to leave this comment.
Maybe not just "former activity" but "ongoing former activity" to be more precise.

File: help/C/merge.page [hunk 3]
```diff
@@ -9,26 +9,18 @@
...
-        day. Start by entering the first activity and select the "in progress"
-        check box for the end time. For the next activity again set just the
-        start time and mark it as ongoing. Observe how the end time of the
-        previous activity gets adjusted to the start time of the new one.
-        Repeat the process until happy!
+        day. Start by entering the first activity with only a start time
+        (i.e. activity "in progress"). For the next activity again set just
+        the start time. Observe how the end time of the previous activity
+        gets adjusted to the start time of the new one. Repeat the process
+        until happy!
```
Comment (ederag) [inline help/C/merge.page]: Indeed, that needed an update, well done !

**Comment by GeraldJansen** [comment]
I'm new to revising merge requests, so I'm not sure how to proceed. I simply clicked on the "Resolve conversation" buttons, hoping that meant accepting your proposals. Anyway, you should have push access to my branch if that did not work.

**Comment by ederag** [comment]
We are both learning then !
"[Resolve conversation](https://help.github.com/en/articles/commenting-on-a-pull-request#resolving-conversations)" just collapses the conversation.
Unfortunately, it is not possible to propose multiple lines changes yet (I already reported this wish to github).
I'll try to push a commit to your branch.

**Comment by ederag** [comment]
Done, thanks for your contribution !

**Pull request merged**